docs/actions.md

```diff
@@ -2,9 +2,16 @@
 # Creating and invoking OpenWhisk actions
 
 
-Actions are stateless code snippets that run on the OpenWhisk platform. An action can be written as a JavaScript, Swift, or Python function, a Java method, or a custom executable program packaged in a Docker container. For example, an action can be used to detect the faces in an image, respond to a database change, aggregate a set of API calls, or post a Tweet.
+Actions are stateless code snippets that run on the OpenWhisk platform.
+For example, an action can be used to detect the faces in an image, respond to a database change,
+aggregate a set of API calls, or post a Tweet.
+An action can be written as a JavaScript, Swift, Python or PHP function, a Java method,
+any binary-compatible executable including Go programs and custom executables packaged as Docker containers.
 
-Actions can be explicitly invoked, or run in response to an event. In either case, each run of an action results in an activation record that is identified by a unique activation ID. The input to an action and the result of an action are a dictionary of key-value pairs, where the key is a string and the value a valid JSON value. Actions can also be composed of calls to other actions or a defined sequence of actions.
+Actions can be explicitly invoked, or run in response to an event.
+In either case, each run of an action results in an activation record that is identified by a unique activation ID.
+The input to an action and the result of an action are a dictionary of key-value pairs, where the key is a string and the value a valid JSON value.
```

Comment:
Regarding this line: `The input to an action and the result of an action are a dictionary of key-value pairs, where the key is a string and the value a valid JSON value.` Isn't all action input and output JSON? Maybe I'm just confused by it saying only the value is JSON.

Reply:
Note: This isn't new text - i just reformatted the text that was there. No semantic change.

JSON is a representation of a dictionary native to JS. In Swift and Python we use a dictionary type that is native to those languages.